src/modules/universal-discovery/components/bookmarks/bookmarks.component.js

```diff
@@ -0,0 +1,174 @@
+     * @memberof BookmarksComponent
+     */
+    removeItem(itemToRemoveLocation) {
+        const areTheSameItems = (fstItemLocation, sndItemLocation) => fstItemLocation.id === sndItemLocation.id;
+
+        this.setState(state => Object.assign({}, state, {
+            items: state.items.filter((item) => !areTheSameItems(item.value.Location, itemToRemoveLocation))
+        }));
+    }
+
+    /**
+     * Creates bookmarks loading error notification
+     *
+     * @method createBookmarksLoadingErrorNotification
+     * @memberof BookmarksComponent
+     */
+    createBookmarksLoadingErrorNotification() {
+        const bookmarksLoadingErrorEvent = new CustomEvent('ez-notify', {
+            detail: {
+                label: 'danger',
+                message: 'Bookmarks loading error.'
+            }
+        });
+
+        document.body.dispatchEvent(bookmarksLoadingErrorEvent);
+    }
+
+    /**
+     * Loads bookmarks
+     *
+     * @method bookmarksContent
+     * @memberof BookmarksComponent
+     */
+    bookmarksContent() {
+        this.setState(state => Object.assign({}, state, { isLoading: true }), () => {
+            const { restInfo } = this.props;
+            const bookmarksLoaded = new Promise(resolve => loadBookmarks(restInfo, resolve));
+
+            bookmarksLoaded
+                .then(this.updateItemsState)
+                .catch(this.createBookmarksLoadingErrorNotification);
+        });
+    }
+
+    /**
+     * Updates items state with bookmarks results
+     *
+     * @param {Object} response content query REST endpoint response
+     * @memberof BookmarksComponent
+     */
+    updateItemsState(response) {
+        this.setState(state => Object.assign({}, state, {
+            items: response.View.Result.searchHits.searchHit,
+            isLoading: false
+        }));
+    }
+
+    getTableLabels() {
+        const {
+            tableTitle,
+            paginationFirst,
+            paginationPrev,
+            paginationNext,
+            paginationLast,
+            tableHeaderName,
+            tableHeaderType,
+            tableItemNotAvailable
+        } = this.props.labels.bookmarks;
+
+        return {
+            title: tableTitle,
+            header: {
+                name: tableHeaderName,
+                type: tableHeaderType
+            },
+            item: {
+                notAvailable: tableItemNotAvailable
+            },
+            pagination: {
+                first: paginationFirst,
+                prev: paginationPrev,
+                next: paginationNext,
+                last: paginationLast,
+            }
+        };
+    }
+
+    render() {
+        const { onItemSelect, bookmarksPerPage, contentTypesMap, maxHeight } = this.props;
+        const tableLabels = this.getTableLabels();
+
+        return (
+            <div className="c-bookmarks" style={{ maxHeight: `${maxHeight - 32}px` }}>
+                <div className="c-bookmarks__results">
```

Comment:
Do you need it? I don't see it's styled anywhere nor referenced in the JS code, but I might be missing something.